package pb;

message Frame {
    required uint32       timestamp_unix = 1;
    required uint32       timestamp_s   = 2;
    required uint32       timestamp_us  = 3;
    required bytes        blob          = 4;
}
